demo-app/src/open-truss/configs/11-rest-demo.yaml

```diff
@@ -0,0 +1,32 @@
+workflow:
+  version: 1
+  signals:
+    repo_nwo_input: string
+    results:
+      total_count: number
+      incomplete_results: boolean
+  frames:
+    - frame:
+      view:
+        component: OTRestDataProvider
+        props:
+          _DEBUG_: true
+          source: rest-demo
+          path: /search/issues?q=repo:${repo_nwo}+is:issue
+          path_values:
```

Comment:
I'm torn between the two. I like separate props, but I also like the approach of encapsulating everything into a single `path` object. `path` could be a string literal (if it's hardcoded) or a template object (if it's parameterized). What do you think @jonmagic?

That said, if we do opt for the object config: I'm really averse to using `_` prefixes for normal APIs. It makes sense for system APIs (like `_DEBUG_`) but if we adopt it for public APIs I think it makes the YAML much harder to read. So I'd want to just use `template` as the "keyword" instead of `_template`.

Reply:
👍 to encapsulating in a single path object but I agree that the template key should just be `template` and not have the `_` prefix.

So you would either hard code a path with no variables like this:

```yaml
path: /this/path/has/no/interpolation
```

Or interpolate variables like this:

```yaml
path:
  template: /this/${one}/${does}
  one: :foo
  does: :bar
```

Reply:
@andrew-dillon (and @jonmagic since you authored the other data provider), which it makes sense it would be great to have docs about how to use these two components and their configs ❤️

Reply:
Pushed a commit replacing `path_values` with a template object shape option for `path` 👍